### ISSUE TITLE
Display Specific Flickering / Jumping Issue from 6.0.0 > 7.0.0

### ISSUE THREAD

**Issue by aaronbarnaby** [konvajs/konva#938]
Since updating from 6.0.0 to 7.0.0 without any change to code I've noticed a large breaking issue to my application that only effects certain displays.

I have 2 monitors on my standard HD monitor running at 1920x1080 I have no issue at all and everything runs as before.

When switching over to my 4K monitor running 3840 x 2160 I start seeing issues.
The issues I have are as follows:

when dragging a 'Image' shape you can see visible flickering of the image (and not just display and not displaying) the image flickers in positional coords of the canvas.
this triggers problem two which is transforming a image shape,  when dragging the corner (seems to effect the top left corner mostly) the image will just jump out of canvas.
I'm guessing whats happening is it's 'flickering' and because i'm still mouse down that's detecting a coords change.

This happens also on a Mac device (getting cause of the high resolution also) and does fix the issue if i change the display to run in 1920x1080.

Does anyone know anything about this issue or have they experienced it themselves?

I'm going to look though the changed code of the library between 6.0.0 > 7.0.0 because it's obviously something there that's causing it, I'll report back if I find anything.

**Comment by lavrton** [comment]
Can you make a demo?

**Comment by lavrton** [comment]
You may try to log `{x, y}` position:

```js
image.on('xChange yChange', () => {
  console.log(image.x(), image.y());
});
```

Do you see any crazy numbers?

**Comment by aaronbarnaby** [comment]
I didn't think i would be able to get a working demo however seems like i can.

https://jsbin.com/xufabizeqo/edit?js,console,output

It doesn't seem to cause the visible flickering issue because that will be caused by my code making it visibly flicker when the numbers are messing up.

`
"x: 151.12477214367178   y: 68.30425867591454"
"x: 151.12477214367178   y: 68.77039187922696"
"x: 4.999999999999972   y: 68.77039187922696"
"x: 4.999999999999972   y: 5"
"x: 131.97584759212393   y: 5"
"x: 131.97584759212393   y: 91.9697586247425"
"x: 133.02653320979095   y: 91.9697586247425"
"x: 133.02653320979095   y: 92.6894063080761"
`

as you can see here i caused the image to jump to different coords I did this by moving the top-left transform control
it seems to only happen at the start of dragging so the ways to replicate are:

1. Click top-left transform control
2. Start dragging
3. IF images doesn't jump release dragging and repeat step 1

And to confirm this only happens on a high resolution display.
I can repeat the issue on my 4K display but on my standard HD display it never happens.

**Comment by lavrton** [comment]
I see the bug. Will check it.

**Comment by lavrton** [comment]
Should be fixed with `konva@7.0.1`. Please try it. Let me know if it helps.

**Comment by aaronbarnaby** [comment]
Thanks for this works perfectly again now 👍
